master/docs/manual/cfg-statustargets.rst

```diff
@@ -1,55 +1,49 @@
-.. bb:cfg:: status
+.. bb:cfg:: reporter
 
-.. _Status-Targets:
+.. _Reporters:
 
-Status Targets
---------------
+Reporters
+---------
 
 .. contents::
     :depth: 2
     :local:
 
 The Buildmaster has a variety of ways to present build status to various users.
-Each such delivery method is a `Status Target` object in the configuration's :bb:cfg:`status` list.
-To add status targets, you just append more objects to this list::
+Each such delivery method is a `Reporter Target` object in the configuration's ``services`` list.
+To add reporter targets, you just append more objects to this list::
 
-    c['status'] = []
+    c['services'] = []
```

Comment:
Hmm, we already have support for this key via #1378, so this section should reference that documentation.  Maybe the wording's just a bit confusing here.
```
Each is a Buildbot service (see :bb:cfg:`services`) which communicates with the rest of Buildbot to learn about build status.
```